.github/workflows/release.yml

```diff
@@ -132,6 +132,10 @@ jobs:
             --output-signature spin.sig \
             ${{ matrix.config.targetDir }}/spin${{ matrix.config.extension }}
 
+      - name: Generate manifest schema
+        if: matrix.config.os == 'ubuntu-22.04' && matrix.config.arch == 'amd64'
```

Comment:
It might be nice to make a new matrix property, either specific like `generateManifestSchema: true` or generic `doOneTimeStuff`. I'm 90% sure if you omit a matrix property it will evaluate falsey in this context.